src/PowerShellEditorServices/Services/PowerShellContext/Handlers/PSHostProcessAndRunspaceHandlers.cs

```diff
@@ -17,12 +17,12 @@ namespace Microsoft.PowerShell.EditorServices.Handlers
 
     internal class PSHostProcessAndRunspaceHandlers : IGetPSHostProcessesHandler, IGetRunspaceHandler
     {
-        private readonly ILogger<GetVersionHandler> _logger;
+        private readonly ILogger<PSHostProcessAndRunspaceHandlers> _logger;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unread private field '_logger' or refactor the code to use its value.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=5596138)